pyproject.toml

```diff
@@ -5,5 +5,4 @@ requires = [
   "setuptools_scm_git_archive >= 1.0",
   "wheel",
 ]
-requires-python = ">=3.8"
```

Comment:
this triggerred failing builds as it was an unknown parameter

Reply:
my bad, sorry about that

Reply:
Just looked into this, I think this needs to be under the `[project]` header:
https://stackoverflow.com/a/72462723/7724187

Reply:
also found that but there's like much inclarity as to what does things actually do ... many people are confused :-D as am I.

I'll follow your suggestion

Reply:
didnt work :-S